tests/e2e/Services/Migrations/MigrationsBase.php

```diff
@@ -579,6 +589,21 @@ public function testAppwriteMigrationDatabasesDocument(array $data)
             'apiKey' => $this->getProject()['apiKey'],
         ]);
 
+        // Wait for usage dump
+        sleep(30);
+
+        $finalStats = $this->client->call(Client::METHOD_GET, '/project/usage', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), [
+            'startDate' => UsageTest::getYesterday(),
+            'endDate' => UsageTest::getTomorrow(),
+        ]);
+
+        // Ensure database reads/writes did not change
```

Comment:
Is any usage stat affected by this? If so, let's have check that it assertNotEquals. That will give us asurance that usage dump cycle ran.